websites/management/commands/unpublish_sites.py

```diff
@@ -0,0 +1,124 @@
+                    (
+                        Q(metadata__courses__icontains=website.name)
+                        | Q(metadata__resources__content__icontains=website.name)
+                    ),
+                    website__name=settings.ROOT_WEBSITE_NAME,
+                )
+                course_content_dependencies = WebsiteContent.objects.filter(
+                    ~Q(website__name=website.name),
+                    type=CONTENT_TYPE_PAGE,
+                    markdown__icontains=website.name,
+                )
+                course_dependencies = Website.objects.filter(
+                    ~Q(name=website.name), metadata__icontains=website.name
+                )
+                # if there are dependencies, don't unpublish and list dependencies
+                if (
+                    ocw_www_dependencies
+                    or course_dependencies
+                    or course_content_dependencies
+                ):
+                    self.stdout.write(
+                        "Not unpublishing " + website.name + " due to dependencies:"
+                    )
+                    if ocw_www_dependencies:
+                        self.stdout.write(
+                            json.dumps(
+                                {
+                                    "ocw_www": WebsiteContentSerializer(
+                                        instance=ocw_www_dependencies, many=True
+                                    ).data
+                                }
+                            )
+                        )
+                    if course_dependencies:
+                        self.stdout.write(
+                            json.dumps(
+                                {
+                                    "course": WebsiteBasicSerializer(
+                                        instance=course_dependencies, many=True
+                                    ).data
+                                }
+                            )
+                        )
+                    if course_content_dependencies:
+                        self.stdout.write(
+                            json.dumps(
+                                {
+                                    "course_content": WebsiteContentSerializer(
+                                        instance=course_content_dependencies, many=True
+                                    ).data
+                                }
+                            )
+                        )
+                else:  # unpublish
+                    unpublished_count += 1
+                    unpublished_names.append(website.name)
+                    Website.objects.filter(pk=website.pk).update(
+                        unpublish_status=PUBLISH_STATUS_NOT_STARTED,
+                        last_unpublished_by=user_id,
+                    )
+                    site_pipeline = api.get_site_pipeline(website)
+                    site_pipeline.pause_pipeline(VERSION_LIVE)
```

Comment:
`VERSION_LIVE` needs to be in imports

Reply:
Updated in aae71ff0591b74a5b003e18a400de7b510d188c8.